src/main/scala/com/github/mrpowers/spark/fast/tests/DataframeUtil.scala

```diff
@@ -8,8 +8,8 @@ object DataframeUtil {
 
   private[mrpowers] def showDataframeDiff(
```

Comment:
we should probably add tests for either `showDataframeDiff` or the public assert API to make sure the displayed content are colored correctly